CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,136 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
```

Comment:
Do we need to define "community"? This is a repo but also an organization, and I'd want the code to apply to the entire set of HoloViz repos.

Reply:
ok, I added a community definition in the 'scope' section. Feel free to edit

Reply:
Looks perfect, thanks!